src/App.css

```diff
@@ -70,6 +70,35 @@ body {
   list-style: none;
 }
 
+.card-title a {
+  text-decoration: none;
+  color: #022b02;
+  font-size: 1em;
+  transition: all .3s ease;
+}
+
+.card-title a:hover {
+  text-decoration: none;
+  color: #077007;
+  font-size: 1.005em;
+}
+
+.an-article {
+  gap: 30px;
+  min-width: 500px;
```

Comment:
Check out what this looks like on a mobile screen.

Reply:
Unable to resolve. Will reformat design at a later stage.